sdv/lite/tabular.py

```diff
@@ -251,7 +272,11 @@ def load(cls, path):
             TabularModel:
                 The loaded tabular model.
         """
-        return cls._default_model.load(path)
+        with open(path, 'rb') as f:
+            model = pickle.load(f)
+            throw_version_mismatch_warning(getattr(model, '_package_versions', None))
+
+            return
```

Comment:
Should this return the model?

Reply:
Yes, good catch. I must've deleted it by accident.. updated the unit test as well to test this.